### ISSUE TITLE
Run button does nothing when app has not been built yet

### ISSUE THREAD

**Issue by Hofer-Julian** [bilelmoussaoui/flatpak-vscode#134]
When the triangle "run" button is being pressed without building the app first nothing happens.
Ideally it would build and then run the app like Builder does, but if that's hard to do a warning would suffice as well.

**Comment by SeaDve** [comment]
Indeed, the issue is being tracked in #63 and #73. I'm thinking of improving the pipeline system to be more extensible and simpler.

**Comment by Hofer-Julian** [comment]
> Indeed, the issue is being tracked in #63 and #73. 

I see, feel free to close this issue then if you feel like its already covered :)

**Comment by SeaDve** [comment]
Adding a warning message would be great though, and close this once it is merged
